### ISSUE TITLE
Make local stitchit ini doesn't work with the the new _MACHINE files

### ISSUE THREAD

**Issue by raacampbell** [BaselLaserMouse/StitchIt#70]
I make the local stititch ini and it doesn't have the _MACHINE name. So StitchIt continues to load the general one.

**Comment by ablot** [comment]
? I'm pretty sure I don't have this bug. Can you reproduce it on linux machines?

**Comment by raacampbell** [comment]
It was making the file using the default empty INI. It now no longer is doing so: it's using the current system default.

**Comment by raacampbell** [comment]
The bug persists, though:

```
>> [~,p]=readStitchItINI

p =

    '/opt/matlabCommon/stitchitConf_brainsaw.ini'

>> ls *.ini
stitchitConf.ini

>> pwd

ans =

    '/mnt/data/BakingTrayStacks/ERAE28_2c'
```

**Comment by raacampbell** [comment]
Fixed b6f9af4b12ae7defb693d938b52018c1253e62e4
